src/AutoRest.CSharp/Common/Generation/Writers/ModelWriter.cs

```diff
@@ -288,13 +288,48 @@ private void BuildHierarchy(ObjectTypeProperty property, Stack<ObjectTypePropert
 
         private FormattableString CreatePropertyDescription(ObjectTypeProperty property, string? overrideName = null)
         {
+            FormattableString binaryDataExtraDescription = Configuration.AzureArm &&
```

Comment:
remove the if arm check I believe this should apply to all libraries

Reply:
removed